packages/sdk/src/browser-wallets/magiceden/index.ts

```diff
@@ -19,33 +20,49 @@ export interface MagicEdenWindow extends Window {
   BitcoinProvider?: MagicEdenBitcoinProvider;
 }
 
+export interface MagicEdenWallet extends Wallet {
+  name: "Magic Eden";
+  features: {
+    "sats-connect:": {
+      provider: MagicEdenBitcoinProvider;
+    };
+  };
+}
+
+async function getMagicEdenWalletProvider(): Promise<BitcoinProvider> {
+  const meWallet = getWallets()
+    .get()
+    .find((wallet) => wallet.name === "Magic Eden");
```

Comment:
`.find((wallet) => wallet.name === "Magic Eden" && "sats-connect:" in wallet.features);`

There are two wallets entries for magic eden with the same name, we should check that sats-connect is present for that wallet.

Reply:
Fixed